libs/web-components/src/components/dropdown/Dropdown.svelte

```diff
@@ -66,6 +66,9 @@
   export let error: string = "false";
   export let multiselect: string = "false";
   export let native: string = "false";
+  export let size: "regular" | "compact" = "regular";
```

Comment:
My mistake in the suggestion, but to be consistent with other components lets rename "regular" to "default"


```suggestion
  export let size: "default" | "compact" = "default";
```